metaflow/plugins/aws/batch/batch_decorator.py

```diff
@@ -322,6 +310,21 @@ def task_pre_step(
 
         if num_parallel >= 1:
             _setup_multinode_environment()
+            # current.parallel.node_index will be correctly available over here.
+            meta.update({"parallel-node-index": current.parallel.node_index})
```

Comment:
same change would be needed in the kubernetes decorator too - maybe it is in the other PR

Reply:
its in another PR which changes k8s decorator.